src/schema/nmdc.yaml

```diff
@@ -1602,6 +1602,16 @@ enums:
       Metagenome Raw Reads:
         annotations:
           file_name_pattern: '/.+?(?=fastq)/fastq.gz'
+      Metagenome Raw Read 1:
```

Comment:
- I will add clarification about R1 = fwd, R2 = rev
- I will update the pattern annotations like ^.+R1\.fastq(\.gz)?$
- in a separate issue, we should clarify how the pattern annotations are used and whether they need to be checked more globally